sample.py

```diff
@@ -18,9 +18,9 @@
 # message_number_graph_day_list = network.message_number_graph(log_data, nicks, nick_same_list, DAY_BY_DAY_ANALYSIS = True)
 #degree_anal_message_numder = network.degree_analysis_on_graph(message_number_graph)
 # message_time_graph_list = network.create_message_time_graph(log_data, nicks, nick_same_list)
-out_degree_node_number, in_degree_node_number, total_degree_node_number = network.degreeNodeNumberCSV(log_data, nicks, nick_same_list)
+#out_degree_node_number, in_degree_node_number, total_degree_node_number = network.degreeNodeNumberCSV(log_data, nicks, nick_same_list)
 #nick_change_graph_list =  user.nick_change_graph(log_data)
-
+bin_matrix, ans = network.create_message_number_binsCSV(log_data, nicks, nick_same_list)
```

Comment:
ans is not a good variable name here.
also i think it'll be better if we skip the word create in these functions which the user will use.
I mean this is redundant right? obviously it will create the message number bins.
i feel `message_number_binsCSV` is better naming. what do you think?